ix-dev/community/palworld/questions.yaml

```diff
@@ -53,6 +53,15 @@ questions:
                   type: string
                   private: true
                   default: ""
+        - variable: allowPlatform
+          label: Platform
+          description: |
+                  Specify which platforms are allowed to connect
+                  Steam or Xbox (Default value is Steam)
+          schema:
+            type: string
+            required: true
+            default: "Steam"
```

Comment:
Lets make this an enum, assuming that its only those 2 values!

Reply:
Added the Enum